solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/util/FlowReceiverContainer.java

```diff
@@ -58,29 +50,22 @@ public class FlowReceiverContainer {
 	private final UnsignedCounterBarrier unacknowledgedMessageTracker = new UnsignedCounterBarrier();
```

Comment:
Similar to `readWriteLock` and `bindCondition` fields, since we've removed `rebind()` (and its corresponding call to `unacknowledgedMessageTracker.awaitEmpty()`), this field can now also be deleted.